pkg/aws_config_server/list_roles.go

```diff
@@ -219,3 +222,13 @@ func GetActiveAccountList(
 
 	return activeAccounts, nil
 }
+
+func getAcctAlias(ctx context.Context, svc iamiface.IAMAPI) (string, error) {
+	input := &iam.ListAccountAliasesInput{}
+	output, err := svc.ListAccountAliases(input)
+	if err != nil {
+		return "", errors.Wrap(err, "Error getting account alias")
+	}
```

Comment:
https://github.com/chanzuckerberg/aws-oidc/blob/e8048d0b3aeb2eced8427a59346773c81f253309/pkg/aws_config_server/list_roles.go#L85-L88
You could do something like that so we don't crash with access denied. We also have logrus.Error hooked up to Sentry so that these don't go unnoticed.